lang/en.json

```diff
@@ -521,135 +521,154 @@
-        "MatomoTagDescription": "Matomo is the leading open source analytics platform.",
-        "MatomoTagHelp": "This tag lets you track pageviews and events into your Matomo. You can customize the tracking to your needs.",
+        "MatomoConfigurationMatomoCustomDimensionsException": "The value needs to be an array",
+        "MatomoConfigurationMatomoBundleTrackerTitle": "Bundle tracker",
+        "MatomoConfigurationMatomoBundleTrackerDescription": "By bundling the Matomo JavaScript tracker directly into the container it may improve your website performance, as it reduces the number of needed requests. It is recommended to bundle the Matomo tracker because in most cases the tracker would otherwise be loaded in a separate request upon pageview anyway. If you use two different Matomo configurations in one container, the setting of the first configuration used in the first Matomo Tag will be applied to all Matomo tags within one container.",
+        "MatomoConfigurationMatomoRegisterAsDefaultTrackerTitle": "Register As default tracker",
+        "MatomoConfigurationMatomoRegisterAsDefaultTrackerDescription": "Registers the tracker as the default one for the website, so that it receives all commands pushed into the global _paq variable. Useful if you want to use the tracker config from the container with your own _paq.push() JavaScript code.",
+        "MatomoConfigurationMatomoJsEndpointTitle": "Path to JavaScript tracker",
+        "MatomoConfigurationMatomoJsEndpointDescription": "Allows setting the source path of the Matomo JavaScript tracker. If you are not using the \"Bundle tracker\" option.",
+        "MatomoConfigurationMatomoTrackingEndpointTitle": "Target path for tracking requests",
+        "MatomoConfigurationMatomoTrackingEndpointDescription": "Allows setting the target path for tracking requests.",
+        "MatomoTagDescription": "Matomo is the copylefted libre privacy-friendly open source analytics-platform.",
+        "MatomoTagHelp": "This tag lets you track pageviews and events into your Matomo. Adapt the tracking to your needs.",
         "MatomoTagName": "Matomo Analytics",
+        "MetaContentVariableName": "Page meta content",
         "MetaContentVariableDescription": "Reads a value from a \"meta\" HTML element.",
-        "MetaContentVariableName": "Page Meta Content",
         "NameOfLatestVersion": "The name of the previous version is %s.",
         "Names": "Names",
         "NoContainersFound": "No container found.",
-        "NoMatomoConfigFoundForContainer": "No Matomo configuration found in this container.",
+        "NoMatomoConfigFoundForContainer": "Could not find any Matomo configuration in this container.",
         "NoReleasesFound": "No release found.",
-        "NoReleasesFoundForContainer": "No release has been found for this container.",
+        "NoReleasesFoundForContainer": "Could not find any release in this container.",
```

Comment:
I think the older text is more appropriate since release is for a container and not a sub part of it

Reply:
```suggestion
        "NoReleasesFoundForContainer": "Could not find any releases in this container.",
```